lib/dor/services/client/workspace.rb

```diff
@@ -46,19 +48,6 @@ def cleanup(workflow: nil, lane_id: nil)
           raise_exception_based_on_response!(resp, object_identifier)
         end
 
-        # After an object has been copied to preservation the workspace can be
-        # reset. This is called by the reset-workspace step of the accessionWF
-        # @param [String] workflow (nil) which workflow to callback to.
-        # @param [String] lane_id for prioritization (default or low)
-        # @raise [UnexpectedResponse] if the request is unsuccessful.
-        # @return nil
-        def reset(workflow: nil, lane_id: nil)
-          resp = connection.post do |req|
-            req.url with_query_params("#{workspace_path}/reset", workflow, lane_id)
-          end
-          raise_exception_based_on_response!(resp, object_identifier) unless resp.success?
-        end
```

Comment:
Leveraging the "delete" call to invoke the reset endpoint in DSA, which will now also do the cleanup.  The reason it is called "reset" is because that is the job in DSA that will remain, because it matches the accessionWF:reset-workspace step which will also remain.  Makes it easier for them all to be sync name wise.